.eslintrc.js

```diff
@@ -70,6 +80,7 @@ module.exports = {
         '@typescript-eslint/require-array-sort-compare': 'off',
         '@typescript-eslint/restrict-plus-operands': 'off',
         '@typescript-eslint/restrict-template-expressions': 'off',
+        '@typescript-eslint/sort-type-union-intersection-members': 'off',
```

Comment:
Disabled this as most of the time it's enforcement simply for enforcement's sake. Don't see much real-world value in this.